src/Makefile

```diff
@@ -278,6 +274,14 @@ ifneq ($(OS), WINNT)
 	@ln -sf libjulia-debug.$(JL_MAJOR_MINOR_SHLIB_EXT) $(build_shlibdir)/libjulia-debug.$(JL_MAJOR_SHLIB_EXT)
 	@ln -sf libjulia-debug.$(JL_MAJOR_MINOR_SHLIB_EXT) $(build_shlibdir)/libjulia-debug.$(SHLIB_EXT)
 endif
+ifeq ($(USE_POLLY_ACC), 1)
+# Place libGPURuntime to be accessible under libjulia's RPATH
+ifeq ($(OS), WINNT)
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.dll $(build_shlibdir)
+else
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.$(SHLIB_EXT) $(build_shlibdir)
```

Comment:
You shouldn't need the conditional here, since `$(SHLIB_EXT)` will be `dll` on Windows